modules/Administration/language/en_us.lang.php

```diff
@@ -968,6 +968,7 @@
     'LBL_ELASTIC_SEARCH_INDEX_SCHEDULE_PART_FAIL' => 'Failed to start partial index.',
 
     // SearchWrapper Settings
+    'LBL_SEARCH_INTERFACE' => 'Search Interface',
     'LBL_SEARCH_ENGINE' => 'Search Engine',
     'LBL_SEARCH_ENGINE_HELP' => 'Configure the default search engine used by the search wrapper',
```

Comment:
Might want to update this while your here: https://github.com/salesagility/SuiteCRM/issues/6540#issuecomment-445213085